cli/resolver.rs

```diff
@@ -8,14 +8,14 @@ use deno_core::futures::future::LocalBoxFuture;
 use deno_core::futures::FutureExt;
 use deno_core::ModuleSpecifier;
 use deno_core::TaskQueue;
-use deno_graph::source::NpmPackageReqResolution;
 use deno_graph::source::NpmResolver;
+use deno_graph::source::PackageReqResolution;
```

Comment:
This rename was actually a mistake because this won't be used for deno: resolution. I will revert it later.